test/cpp/thread_manager/thread_manager_test.cc

```diff
@@ -31,8 +31,14 @@
 namespace grpc {
 class ThreadManagerTest final : public grpc::ThreadManager {
```

Comment:
Can I suggest renaming this struct to MockThreadManager since that's what this essentially is?

Reply:
I would also suggest moving out the PerformTest function. The MockThreadManager should be created by the PerformTest function which is external, and it should take `gpr_atm*` arguments for the two things being counted. You can make this test look more gtest-idiomatic in that way.